test/e2e/core.go

```diff
@@ -97,3 +123,29 @@ func logCore(cmds []command, hosts []string, dir, provider string) {
 	}
 	wg.Wait()
 }
+
+func isUsingSystemdKubelet(provider string, hosts ...string) (bool, error) {
```

Comment:
there are only two return points from this function on lines 147 and 150, and neither one returns an error.